verifier/token/api/api.go

```diff
@@ -15,13 +16,15 @@ func NewHTTPAPI(
 	healthReporters []protocol.HealthReporter,
 ) *gin.Engine {
 	router := gin.Default()
```

Comment:
i think the underlying recovery middleware will still be used might have to do.
```suggestion
	router := gin.New()
	router.Use(gin.Logger())
```